docs/developer_info.rst

```diff
@@ -0,0 +1,213 @@
+************************************************
+Developer documentation for the DIANNA project
+************************************************
+
+This chapter lists the tools and practices we typically use during development.
+Most of our choices are motivated in the `NL eScience Center Guide <https://guide.esciencecenter.nl>`__, specifically the `Python chapter <https://guide.esciencecenter.nl/#/best_practices/language_guides/python>`__ and our `software guide checklist <https://guide.esciencecenter.nl/#/best_practices/checklist>`__.
+If you're considering contributing to DIANNA, please have a look and be sure to let us know (e.g. via an issue) if you have any questions.
+
+
+Development install
+-------------------
+
+.. code:: shell
+
+   # Create a virtual environment, e.g. with
+   python3 -m venv env
+
+   # activate virtual environment
+   source env/bin/activate
+
+   # make sure to have a recent version of pip and setuptools
+   python3 -m pip install --upgrade pip setuptools
+
+   # (from the project root directory)
+   # install dianna as an editable package
+   python3 -m pip install --no-cache-dir --editable .
+   # install development dependencies
+   python3 -m pip install --no-cache-dir --editable .[dev]
+
+Afterwards check that the install directory is present in the ``PATH`` environment variable.
```

Comment:
I think you mean `PYTHONPATH`, but even then I don't think it's actually in there. In my experience at least Python creates a link in the `site-package` folder to wherever the original code is. I don't know if it always works like that though

Reply:
Good catch, I'll just take the whole sentence out, seems useless with modern `venv` and `conda`.